e2e/disposing/disposing.spec.ts

```diff
@@ -25,13 +25,26 @@ test('no error on constructing and disposing', async ({ page }) => {
     });
 
     await page.goto('http://localhost:3000/sheets/');
-    await page.waitForTimeout(2000);
-
     await page.evaluate(() => window.E2EControllerAPI.loadDefaultSheet());
-    await page.waitForTimeout(6000); // wait for features loaded in STEADY stage (5s)
+    await page.evaluate(() => window.E2EControllerAPI.disposeUniver());
+    expect(errored).toBeFalsy();
+});
 
+test('no error on constructing and disposing sheet unit', async ({ page }) => {
+    // test.setTimeout(0); // Keep wait result forever
+
+    let errored = false;
+    page.on('pageerror', (error) => {
+        console.error('Page error:', error);
+        errored = true;
+    });
+
+    await page.goto('http://localhost:3000/sheets/', { waitUntil: 'networkidle' });
+    await page.evaluate(() => window.E2EControllerAPI.disposeDefaultSheetUnit());
+    await page.evaluate(() => window.E2EControllerAPI.loadDefaultSheet());
```

Comment:
可以试试延时不同的时间

Reply:
Since loadDefaultSheet is already awaited, an external await is unnecessary.
Maybe loadDefaultSheet should not await.
<img width="1187" alt="image" src="https://github.com/user-attachments/assets/3d6fe0be-cf61-46c1-8dd5-e542660de30a">